### PR TITLE
Added gitter channel link

### LINKED ISSUE #159: Add the gitter channel
Adding gitter channel link to the contribute page of circuit verse

### PR DESCRIPTION
**Pull request by creyente1897**
Fixes #159 

screenshot from 2019-03-02 03-00-33

### CODE REVIEW

**Comment by shubhankarsharma00** [comment]
> Issue #160
I think you mean issue #159

File: app/views/logix/contribute.html.erb [hunk 1]
```diff
@@ -15,6 +15,10 @@
                         Contact us @
                         <a href="mailto:support@circuitverse.org" target="_top">support@circuitverse.org</a>
                       </p>
+                      <p>
+                        Join our chat room @
+                        <a href="https://gitter.im/CircuitVerse/community" target="_top">Gitter Channel</a>
```
Comment (armudgal) [inline app/views/logix/contribute.html.erb]: Can we open the chat room in a new tab using `_blank`. Thanks

Comment (creyente1897) [reply app/views/logix/contribute.html.erb]: Yes sir will do so
Thanks for pointing out the issue number will change it too :+1:

Comment (armudgal) [reply app/views/logix/contribute.html.erb]: Hi, no sir please. Haven't received my knighthood (yet).

Comment (creyente1897) [reply app/views/logix/contribute.html.erb]: @armudgal Ok :smile:
I have also included the chat link in the about page.

**Comment by armudgal** [comment]
Hey, before I merge this PR can you please squash all the commits

**Comment by armudgal** [comment]
Hi. i can still see 5 commits,  to squash your last 5 commits `git rebase -i HEAD~5` and replace `pick` with `squash` for the commits you want to squash.
Once you are done, with rebasing, `git push` might not work and you might need to do `git push --force`. 
Just make sure before all this that you are on the correct branch

**Comment by creyente1897** [comment]
> Hi. i can still see 5 commits, to squash your last 5 commits `git rebase -i HEAD~5` and replace `pick` with `squash` for the commits you want to squash.
> Once you are done with rebasing, `git push` might not work and you might need to do `git push --force`.
> Just make sure before all this that you are on the correct branch.
> 
> PS: [How-to-squash](https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit) text might help

I followed your instructions and squashed the commits .
Can you please check :smile: 
I am sorry for the delay

**Comment by armudgal** [comment]
Thanks, cheers :cake:

**Pull request merged**